docs/sdk/wrapped-keys/sign-transaction.md

```diff
@@ -0,0 +1,607 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Signing a Transaction
+
+This guide covers the `signTransactionWithEncryptedKey` function from the Wrapped Keys SDK. For an overview of what a Wrapped Key is and what can be done with it, please go [here](./overview.md).
+
+Using the `signTransactionWithEncryptedKey` function, you can sign an arbitrary message using a Wrapped Key. The Wrapped Keys SDK will look up the corresponding encryption metadata (`ciphertext` and `dataToEncryptHash`) for your PKP in Lit's private DynamoDB instance. If found, it well then use your provided PKP Session Signatures to authorize decryption of the private key, and will sign your provided message, returning the signed message.
```

Comment:
We should add a line about broadcast in the summary here

Reply:
Addressed by [this commit](https://github.com/LIT-Protocol/docs/pull/302/commits/651f6c5616424bf12cbb6ee80cba88e6201393db)